pykern/pkunit.py

```diff
@@ -549,7 +548,13 @@ def _message(self):
     """
 
     def _set_expect_and_actual(self):
+        (self._read, self._write) = (
```

Comment:
why parens around ` (self._read, self._write) `. Is that what black does or is it a radiasoft style guideline to have parens around a tuple unpacking assignment?

Reply:
Uh, cuz I had no idea that this works: `a, b = 1, 2`. Note that the other parens are necessary due to the inline `if`. 

Fixed a8fd141